org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/src/main/java/org/eclipse/che/jdt/ls/extension/api/dto/ResourceToMove.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.jdt.ls.extension.api.dto;
+
+/**
+ * Represents java compilation unit or package.
+ *
+ * @author Valeriy Svydenko
+ */
+public class ResourceToMove {
```

Comment:
Why not pass a resource here? Should be one of the generalized structural elements.